src/app/components/elements/Voting.jsx

```diff
@@ -398,7 +398,14 @@ class Voting extends React.Component {
                 !(is_comment && total_votes == 0));
         const payoutItems = [];
 
+        const minimumAmountForPayout = 0.02;
+        let warnZeroPayout = '';
+        if (pending_payout > 0 && pending_payout < minimumAmountForPayout) {
+            warnZeroPayout = tt('voting_jsx.must_reached_minimum_payout');
+        }
+
         if (cashout_active) {
+            const payoutDate =  <span>{tt('voting_jsx.payout')} {' '} <TimeAgoWrapper date={cashout_time} /></span>;
```

Comment:
This creates two nested SPAN containers but that's the only way I've found to concatenate "Payout " before the TimeAgo...